src/main/java/seedu/address/logic/commands/RemoveLabCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_LAB;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.lab.Lab;
+
+/**
+ * Deletes a Lab identified using its LAB_NUMBER from the TAddressBook.
+ */
+public class RemoveLabCommand extends Command {
+
+    public static final String COMMAND_WORD = "labrm";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Removes the Lab identified by the LAB_NUMBER from the TAddressBook.\n"
+            + "Parameters: "
+            + PREFIX_LAB + "LAB_NUMBER \n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_LAB + "1";
+
+    public static final String MESSAGE_SUCCESS = "Lab removed";
```

Comment:
Perhaps `MESSAGE_SUCCESS` should include the lab number that was removed too? (e.g. Lab xx removed)

Reply:
Sounds good!